client/components/Categories.jsx

```diff
@@ -14,12 +15,7 @@ const ScoreGrid = styled.section`
 
 const Categories = (props) => (
   <ScoreGrid>
-    <Category/>
-    <Category/>
-    <Category/>
-    <Category/>
-    <Category/>
-    <Category/>
+    { _.map(props.ratings, (rating, name) => <Category name={name} rating={rating} />)}
```

Comment:
Is it giving you an error about the absence of a key for the .map function? if each rating has its own unique ID number, you could use that as a key!!